lumicks/pylake/force_calibration/touchdown.py

```diff
@@ -2,6 +2,68 @@
 from scipy.optimize import curve_fit
 
 
+def mack_model(
+    wavelength_nm,
+    refractive_index_medium,
+    nanostage_z_position,
+    surface_position,
+    displacement_sensitivity,
+    intensity_amplitude,
+    intensity_phase_shift,
+    intensity_decay_length,
+    scattering_polynomial_coeffs,
+    focal_shift,
+    nonlinear_shift,
+):
```

Comment:
should we maybe move this to `tests.data.somewhere`? Since it seems like it's only used in the tests?

Reply:
yeah, you're right it'll likely end up back here anyway. keeping it here is fine